cmake/modules/FindEbur128.cmake

```diff
@@ -45,7 +45,8 @@ The following cache variables may also be set:
 
 find_package(PkgConfig QUIET)
 if(PkgConfig_FOUND)
-  pkg_check_modules(PC_Ebur128 QUIET libebur128>=1.2.4)
+  # The bundled version 1.2.6 only fixes dynamic linking on Windows
+  pkg_check_modules(PC_Ebur128 QUIET libebur128>=1.2.5)
```

Comment:
Is the version check correct here? For my understanding it belongs to the CMakeList.txt file.

Reply:
This would be an unrelated change of the build. Out of scope here.